pmd-apex/src/main/resources/rulesets/apex/metrics.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+
+<ruleset name="Metrics temporary ruleset"
+         xmlns="http://pmd.sourceforge.net/ruleset/3.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/3.0.0 http://pmd.sourceforge.net/ruleset_3_0_0.xsd">
+
+    <description>
+        These are rules which use the Metrics Framework to calculate metrics.
+    </description>
+
+    <rule name="CyclomaticComplexity"
+          message="The {0} ''{1}'' has a{2} cyclomatic complexity of {3}."
+          since="1.03"
```

Comment:
since="6.0.0"
I think, this rule is new - there is only a "StdCyclomaticComplexityRule" available for apex yet.